policy/microos/rke2.fc

```diff
@@ -0,0 +1,22 @@
+# vim: sw=8:ts=8:et
+
+/etc/systemd/system/rke2.*                                          --  gen_context(system_u:object_r:container_unit_file_t,s0)
+/lib/systemd/system/rke2.*                                          --  gen_context(system_u:object_r:container_unit_file_t,s0)
+/usr/local/lib/systemd/system/rke2.*                                --  gen_context(system_u:object_r:container_unit_file_t,s0)
+/usr/bin/rke2                                                       --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
+/usr/local/bin/rke2                                                 --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
+#/var/lib/cni(/.*)?                                                      gen_context(system_u:object_r:container_var_lib_t,s0)
```

Comment:
commented?

Reply:
comments are to retain file comparison parity line-by-line